examples/c/Makefile

```diff
@@ -2,17 +2,18 @@
 OUTPUT := .output
 CLANG ?= clang
 LLVM_STRIP ?= llvm-strip
-BPFTOOL ?= $(abspath ../tools/bpftool)
-LIBBPF_SRC := $(abspath ../libbpf/src)
+BPFTOOL ?= $(abspath ../../tools/bpftool)
+LIBBPF_SRC := $(abspath ../../libbpf/src)
 LIBBPF_OBJ := $(abspath $(OUTPUT)/libbpf.a)
+VMLINUX := ../../vmlinux/vmlinux.h
 # Use our own libbpf API headers and Linux UAPI headers distributed with
 # libbpf to avoid dependency on system-wide headers, which could be missing or
 # outdated
-INCLUDES := -I$(OUTPUT) -I../libbpf/include/uapi
+INCLUDES := -I$(OUTPUT) -I../../libbpf/include/uapi -I$(dir $(VMLINUX))
 CFLAGS := -g -Wall
 ARCH := $(shell uname -m | sed 's/x86_64/x86/')
 
-APPS = minimal bootstrap uprobe kprobe fentry
+APPS = minimal/minimal bootstrap/bootstrap uprobe/uprobe kprobe/kprobe fentry/fentry
```

Comment:
also, let's keep it as just `minimal bootstrap etc`, so that `make minimal` would do what you'd expect.

Reply:
Chatted offline -- will just hardcode for now. Strangely complex to do this in a makefile.